corehq/apps/userreports/models.py

```diff
@@ -760,7 +766,40 @@ def ui_filters(self):
     @property
     @memoized
     def charts(self):
-        return [ChartFactory.from_spec(g._obj) for g in self.configured_charts]
+        configured_charts = deepcopy(self.configured_charts)
+        for chart in configured_charts:
+            if chart['type'] == 'multibar':
+                y_axis_columns = []
+                for y_axis_column in chart['y_axis_columns']:
+                    if isinstance(y_axis_column, dict):
```

Comment:
what case does this if statement handle?

Reply:
Not sure why this is needed. I thought these would always be dicts with two keys, "column_id" being one of them, but some tests have it written in a way that this is just a string with is the column_id.
So this if takes care of those two possibilities when the y_axis_column is represented as a dict or a string

Reply:
What tests were failing?

Reply:
so some tests use config that has y_axis_columns as a string like https://github.com/dimagi/commcare-hq/blob/079bbc6196469f52af88769111b3339588138230/corehq/apps/userreports/tests/data/configs/sample_report_config.json#L48

and then some as dicts
https://github.com/dimagi/commcare-hq/blob/7d1e80eb2ec5fd189468d7026b8897f017ff4256/corehq/apps/app_manager/tests/test_report_config.py#L115

So originally I added only for dict, but then added support for string as well, for example for this test
https://github.com/dimagi/commcare-hq/blob/f9bd6a31bd57c259fdd8d1545ba3dbd33b248907/corehq/apps/app_manager/tests/test_media_suite.py#L201 since it uses the sample report config shared above